src/rust/ide/view/graph-editor/src/lib.rs

```diff
@@ -1599,7 +1597,23 @@ impl GraphEditorModel {
         self.with_edge_map_target(id,|endpoint|endpoint)
     }
 
-    #[allow(dead_code)]
+    // FIXME[WD]: This implementation is slow. Node should allow for easy mapping between Crumbs
+    //            and edges.
```

Comment:
Is there an issue for optimizing that?

Reply:
This is a very low-level thing, it should be done during https://github.com/enso-org/ide/issues/822 . I' adding the link there.